package/sbp_nmea_bridge/src/sbp_nmea_bridge.c

```diff
@@ -342,6 +349,16 @@ int main(int argc, char *argv[])
                              SETTINGS_TYPE_FLOAT,
                              notify_soln_freq_changed,
                              &state);
+
+  pk_settings_register_watch(settings_ctx,
```

Comment:
btw this should be `pk_settings_register` instead of `pk_settings_register_watch` ?

Reply:
Fixing in https://github.com/swift-nav/piksi_buildroot/pull/1205